src/settings/index.js

```diff
@@ -1,89 +1,87 @@
+      <Fragment>
+        <Pane
+          defaultWidth="20%"
+          paneTitle={
+            <Headline tag="h3" margin="none">
+              <FormattedMessage id="ui-circulation.settings.index.paneTitle" />
+            </Headline>
+          }
+          firstMenu={(
+            <PaneBackLink to="/settings" />
+          )}
+        >
+          <NavList>
+            <IfPermission perm="ui-circulation.settings.loan-policies">
+              <NavListItem to={`${path}/loan-policies`}>
+                <FormattedMessage id="ui-circulation.settings.index.loanPolicies" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.circulation-rules">
+              <NavListItem to={`${path}/circulation-rules`}>
+                <FormattedMessage id="ui-circulation.settings.index.circulationRules" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.fixed-due-date-schedules">
+              <NavListItem to={`${path}/fixed-due-date-schedules`}>
+                <FormattedMessage id="ui-circulation.settings.index.fdds" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.other-settings">
+              <NavListItem to={`${path}/checkout`}>
+                <FormattedMessage id="ui-circulation.settings.index.otherSettings" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.staff-slips">
+              <NavListItem to={`${path}/staffslips`}>
+                <FormattedMessage id="ui-circulation.settings.index.staffSlips" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.cancellation-reasons">
+              <NavListItem to={`${path}/cancellation-reasons`}>
+                <FormattedMessage id="ui-circulation.settings.index.requestCancellationReasons" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.notice-templates">
+              <NavListItem to={`${path}/patron-notices`}>
+                <FormattedMessage id="ui-circulation.settings.index.patronNotices" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.notice-policies">
+              <NavListItem to={`${path}/notice-policies`}>
+                <FormattedMessage id="ui-circulation.settings.index.noticePolicies" />
+              </NavListItem>
+            </IfPermission>
+            <IfPermission perm="ui-circulation.settings.request-policies">
+              <NavListItem to={`${path}/request-policies`}>
+                <FormattedMessage id="ui-circulation.settings.index.requestPolicies" />
+              </NavListItem>
+            </IfPermission>
+          </NavList>
+        </Pane>
+        {children}
+      </Fragment>
```

Comment:
I feel like we could consolidate some of the repetition here if we stash the routespaths/permissions/translations in an external array that we could just loop through for generating these lists of links/routes....

Reply:
LOL like how the Settings component we're replacing worked?

Reply:
I think Jeffrey intentionally didn't do that so that the routes would actually be in index.js as per the intention the whole routing for the module be laid out there.

Reply:
...so we can have our cake and eat it too: just export the routes from index. There's still a layer of indirection by splitting it out into an object but I agree that's better than the extent of duplication.